superset/tasks/schedules.py

```diff
@@ -97,6 +97,16 @@
     ],
 )
 
+AlertContent = namedtuple(
```

Comment:
let's make it typed e.g. https://github.com/apache/incubator-superset/blob/08358d623b4938956526df840ef9e466bf281b6a/tests/sqla_models_tests.py#L141